pass/provider.go

```diff
@@ -24,19 +26,41 @@ func Provider() *schema.Provider {
 		ResourcesMap: map[string]*schema.Resource{
 			"pass_password": passPasswordResource(),
 		},
+		Schema: map[string]*schema.Schema{
+			"prefix": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Global key prefix for gopass provider. Must end with a forward slash. Can be used to specify a mount point.",
+			},
+		},
 	}
 }
 
+func (pp *passProvider) GetPath(data *schema.ResourceData) string {
+	if id := data.Id(); id != "" {
+		return id
+	}
+
+	return pp.prefix + data.Get("path").(string)
+}
+
 func providerConfigureContext(ctx context.Context, data *schema.ResourceData) (interface{}, diag.Diagnostics) {
 	store, err := api.New(ctxutil.WithShowParsing(ctx, false))
 
 	if err != nil {
 		return nil, diag.FromErr(err)
 	}
 
+	// normalize prefix value
+	prefix := data.Get("prefix").(string)
+	if len(prefix) > 0 && (prefix == "/" || prefix[len(prefix)-1:] != "/") {
+		return nil, diag.FromErr(errors.New("the value of prefix must be a string with trailing forward slash"))
```

Comment:
technically speaking, the prefix does not need to end with a slash, and without this check prefix would act as a literal prefix, but I think this adds a bit too much complexity on the user's end. So this error is in good faith to prevent certain use cases with a greater goal of simpler use.

Reply:
I agree, I think this makes sense :)